packages/browser/frontend/sirius-components-browser/src/modelbrowser/ModelBrowserTreeView.tsx

```diff
@@ -53,17 +62,106 @@ export const ModelBrowserTreeView = ({
     maxDepth: 1,
   });
 
+  const { getTreePath, loading: treePathLoading, data: treePathData } = useTreePath();
+  const {
+    getExpandAllTreePath,
+    loading: expandAllTreePathLoading,
+    data: expandAllTreePathData,
+  } = useExpandAllTreePath();
+
+  useEffect(() => {
+    if (!expandAllTreePathLoading) {
```

Comment:
We can remove at least two "if" from this function without a change in behavior, it would improve the readability

Reply:
Done